lisp/modules/rename_cues/rename_ui.py

```diff
@@ -0,0 +1,299 @@
+        self.outRegexLine.setToolTip(
+            translate('RenameCues', 'What you type here is used to rename all cues.\n'
+                                    'The regex line below is used to match patterns, and to '
+                                    'capture parts of the text with parenthesis.\n'
+                                    'You can use the content of first () here with $0, the second with $1, etc..'))
+        self.outRegexLine.textChanged.connect(self.onOutRegexChanged)
+        self.layout().addWidget(self.outRegexLine, 3, 3)
+
+        self.regexLine = QLineEdit()
+        self.regexLine.setPlaceholderText(translate('RenameCues', 'Type your regex here :'))
+        self.regexLine.setToolTip(translate('RenameCues', 'Match patterns with standard python regex.\n'
+                                            'Exemple : [a-z]([0-9]+) will find a lower case character [a-z] '
+                                            'followed by one or more number.\n'
+                                            'Only the numbers will be captured for use with $0 above.\n'
+                                            'For more information, consult python documentation'))
+        self.regexLine.textChanged.connect(self.onRegexLineChanged)
+        self.layout().addWidget(self.regexLine, 4, 3)
+
+        # OK / Cancel buttons
+        self.dialogButtons = QDialogButtonBox()
+        self.dialogButtons.setStandardButtons(QDialogButtonBox.Ok |
+                                              QDialogButtonBox.Cancel)
+        self.layout().addWidget(self.dialogButtons, 6, 3)
+
+        self.dialogButtons.accepted.connect(self.accept)
+        self.dialogButtons.rejected.connect(self.reject)
+
+    def get_selected_cues(self):
+        for cue in Application().layout.get_selected_cues():
+            self._cues_list.append({
+                'cue_name': cue.name,
+                'cue_preview': cue.name,
+                'selected': True,
+                'regex_groups': [],
+                'id': cue.id
+                })
+
+        self.update_preview_list()
+
+    def get_all_cues(self):
+        for cue in Application().cue_model.filter():
+            self._cues_list.append({
+                'cue_name': cue.name,
+                'cue_preview': cue.name,
+                'selected': True,
+                'regex_groups': [],
+                'id': cue.id
+            })
+
+        self.update_preview_list()
+
+    def update_preview_list(self):
+        if self.previewList.topLevelItemCount() == 0:
+            self.populate_preview_list()
+        else:
+            for i in range(self.previewList.topLevelItemCount()):
+                item = self.previewList.topLevelItem(i)
+                item.setText(0, self._cues_list[i]['cue_name'])
+                item.setText(1, self._cues_list[i]['cue_preview'])
+                if self._cues_list[i]['selected']:
```

Comment:
It seemed more complicated at the moment. But I changed it and I think it is much better now